cupy/core/core.pyx

```diff
@@ -685,18 +705,32 @@ cdef class ndarray:
         newarray._f_contiguous = True
         return newarray
 
-    cpdef ndarray ravel(self):
+    cpdef ndarray ravel(self, order='C'):
         """Returns an array flattened into one dimension.
 
         .. seealso::
            :func:`cupy.ravel` for full documentation,
            :meth:`numpy.ndarray.ravel`
 
         """
-        # TODO(beam2d): Support ordering option
+        # TODO(beam2d): Support K ordering option
```

Comment:
Could you add your name, concatenated by a comma followed by a space?
(The original TODO will only be partially resolved, so you don't have to erase the original name).

```suggestion
        # TODO(beam2d, grlee77): Support K ordering option
```